jprotoc/README.md

```diff
@@ -130,4 +130,43 @@ Using the Jdk8 Protoc generator
 3. Reference the java 8 client stubs like this:
     ```java
     MyServiceGrpc8.GreeterCompletableFutureStub stub = MyServiceGrpc8.newCompletableFutureStub(channel);
-    ```
\ No newline at end of file
+    ```
+    
+Implementing Custom Protoc Options
+==================================
+1. Create a custom option as per https://developers.google.com/protocol-buffers/docs/reference/java-generated#extension.
+For example:
+
+    ```
+    syntax = "proto3";
+    
+    package com.example.proto.options;
+    
+    import "google/protobuf/descriptor.proto";
+    
+    option java_multiple_files = true;
+    option java_outer_classname = "ServerOptionsProto";
+    option java_package = "com.example.proto.options";
+    
+    extend google.protobuf.FileOptions {
+        ServerOptions server = 50621;
+    }
+    
+    message ServerOptions {
+        // Java classname
+        string name = 1;
+    }
+    ```
```

Comment:
Do I have to run the protobuf generator as part of the build process for my protobuf plugin? Do I need to add this to my plugin project's POM?